### PR TITLE
[watermarkstat] Add unit tests for watermarkstat show commands

### PR DESCRIPTION
**Pull request by neethajohn**
<!--
Please make sure you've read and understood our contributing guidelines:
https://github.com/Azure/SONiC/blob/gh-pages/CONTRIBUTING.md

** Make sure all your commits include a signature generated with `git commit -s` **

If this is a bug fix, make sure your description includes "closes #xxxx",
"fixes #xxxx" or "resolves #xxxx" so that GitHub automatically closes the related
issue when the PR is merged.

If you are adding/modifying/removing any command or utility script, please also
make sure to add/modify/remove any unit tests from the tests
directory as appropriate.

If you are modifying or removing an existing 'show', 'config' or 'sonic-clear'
subcommand, or you are adding a new subcommand, please make sure you also
update the Command Line Reference Guide (doc/Command-Reference.md) to reflect
your changes.

Please provide the following information:
-->

**- What I did**
Added unit tests for watermarkstat show commands

**- How to verify it**
```
platform linux2 -- Python 2.7.16, pytest-3.10.1, py-1.7.0, pluggy-0.8.0 -- /usr/bin/python2
cachedir: .pytest_cache
rootdir: /sonic/src/sonic-utilities/tests, inifile: pytest.ini
plugins: cov-2.6.0
collected 10 items                                                                                                                                                                                                                      

tests/watermarkstat_test.py::TestWatermarkstat::test_show_pg_shared_wm PASSED                                                                                                                                                     [ 10%]
tests/watermarkstat_test.py::TestWatermarkstat::test_show_pg_headroom_wm PASSED                                                                                                                                                   [ 20%]
tests/watermarkstat_test.py::TestWatermarkstat::test_show_queue_unicast_wm PASSED                                                                                                                                                 [ 30%]
tests/watermarkstat_test.py::TestWatermarkstat::test_show_queue_multicast_wm PASSED                                                                                                                                               [ 40%]
tests/watermarkstat_test.py::TestWatermarkstat::test_show_buffer_pool_wm PASSED                                                                                                                                                   [ 50%]
tests/watermarkstat_test.py::TestWatermarkstat::test_show_pg_shared_peristent_wm PASSED                                                                                                                                           [ 60%]
tests/watermarkstat_test.py::TestWatermarkstat::test_show_pg_headroom_persistent_wm PASSED                                                                                                                                        [ 70%]
tests/watermarkstat_test.py::TestWatermarkstat::test_show_queue_unicast_persistent_wm PASSED                                                                                                                                      [ 80%]
tests/watermarkstat_test.py::TestWatermarkstat::test_show_queue_multicast_persistent_wm PASSED                                                                                                                                    [ 90%]
tests/watermarkstat_test.py::TestWatermarkstat::test_show_buffer_pool_persistent_wm PASSED                                                                                                                                        [100%]

======================================================================================================= 10 passed in 1.76 seconds =======================================================================================================
```

### CODE REVIEW

File: scripts/watermarkstat [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
```
Comment (jleveque) [inline scripts/watermarkstat:1]: Why change this?

**Review by lguohan — APPROVED** [review]

File: scripts/watermarkstat (hunk 1 above)
Comment (neethajohn) [reply scripts/watermarkstat:1]: Ran into this error in PR tests after #1151 was merged, 
/usr/bin/env: 'python': No such file or directory
https://sonic-jenkins.westus2.cloudapp.azure.com/job/common/job/sonic-utilities-build-pr/2537/console

Had to change the path to fix it

Comment (jleveque) [reply scripts/watermarkstat:1]: That's strange. #1151 shouldn't have had an impact on this. Also, all other executable scripts in the repo still use `#!/usr/bin/env python`. I feel like this change shouldn't be necessary.

Comment (neethajohn) [reply scripts/watermarkstat:1]: Not sure how to fix the build failure. seems to be failing every time utilities wheel is built. But if I retain the docker and run the test again using 'python setup.py test', I don't see the issue

Comment (jleveque) [reply scripts/watermarkstat:1]: It sounds like it may be a PATH difference with the sonic-utilities build job in sonic-build-tools.

Comment (neethajohn) [reply scripts/watermarkstat:1]: I am not executing the command directly. I am using 'show' commands to invoke that script. The same format is used in many of the utilities tests which are invoking modules under 'scripts' path and some of them also use "/usr/bin/env python". I fail to understand what is special about this particular script

**Comment by lguohan** [comment]
what is pending here? can we merge this?

**Comment by jleveque** [comment]
@lguohan: I find it odd that Neetha needed to modify the shebang in this file. I would like to understand this better, as it shouldn't be necessary. If you'd like, we can merge this as-is, and @neethajohn can continue to investigate offline.

**Comment by neethajohn** [comment]
@lguohan, can I go ahead with the merge?

**Comment by jleveque** [comment]
I'm OK with merging this, but I'd like to understand further what changed to require this shebang to change.

**Comment by abdosi** [comment]
@neethajohn Please create PR for 201911

**Comment by jleveque** [comment]
@neethajohn, @lguohan: I believe I found the root cause of the shebang issue. I have opened a PR to fix here: https://github.com/Azure/sonic-utilities/pull/1233

**Pull request merged**